maven-model/pom.xml

```diff
@@ -64,7 +64,7 @@ under the License.
                 <artifactItem>
                   <groupId>org.apache.maven</groupId>
                   <artifactId>maven-api-model</artifactId>
-                  <version>4.0.0-alpha-1-SNAPSHOT</version>
+                  <version>4.0.0-alpha-3-SNAPSHOT</version>
```

Comment:
Given the version is not changed during the release, this will break again.  Would you mind changing to `${project.version}` instead ?

Reply:
How about I change it everywhere where possible to `${project.version}`?

Reply:
I've set it to ${project.version} in all files I touched.